src/libcore/intrinsics.rs

```diff
@@ -702,16 +702,17 @@ extern "rust-intrinsic" {
     /// which is unsafe unless `T` is `Copy`. Also, even if T is
     /// `Copy`, an all-zero value may not correspond to any legitimate
     /// state for the type in question.
-    #[unstable(feature = "core_intrinsics",
-               reason = "intrinsics are unlikely to ever be stabilized, instead \
-                         they should be used through stabilized interfaces \
-                         in the rest of the standard library",
-               issue = "0")]
-    #[rustc_deprecated(reason = "no longer used by rustc, will be removed - use MaybeUninit \
-                                 instead",
-                       since = "1.38.0")]
```

Comment:
We can keep it deprecated, should we not?

Reply:
It's an unstable item (that is intended for use inside rustc anyway), I don't have a strong opinion.

Reply:
Please keep the diff small by not removing the deprecation.

That should also make it easier alter to revert this PR.

Reply:
Actually it is probably best to instead *add* a deprecation mark at the other intrinsic. That most accurately reflects their status.